src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -245,45 +245,73 @@ export async function POST(request: NextRequest) {
           subscriber_id: subscriber.id,
```

Comment:
already having a logger.info above (l244, l238)?

Reply:
Yes this one confused me earlier, I think we just need the one that includes the payload.